src/Razor/test/testassets/blazorwasm/Program.cs

```diff
@@ -8,6 +8,7 @@ public static void Main(string[] args)
         {
             GC.KeepAlive(typeof(System.Text.Json.JsonSerializer));
             GC.KeepAlive(typeof(RazorClassLibrary.Class1));
+            GC.KeepAlive(typeof(Microsoft.Extensions.Logging.ILogger));
```

Comment:
Seems strange we need to do this now if we didn't before. Any ideas what changed that caused this to become unrooted?

Reply:
Oh no wait, I see this is the test project! Ignore me...